docker/docker-compose.yaml

```diff
@@ -32,5 +34,21 @@ services:
     ports:
       - 80:80
       - 443:443
+    networks:
+      - app-network
     labels:
       caddy.email: "devops@brainbot.com"
+
+  vm:
+    image: victoriametrics/victoria-metrics
+    restart: always
+    command: -promscrape.config /config/prometheus.yml
+    ports:
+      - "8428:8428"
+    volumes:
+      - ../data/victoriametrics:/victoria-metrics-data
+      - ../config:/config
```

Comment:
Sorry I think we had a miscommunication here.
We should scrape the metrics from our already existing VictoriaMetrics installation. 
Doesn't really make too much sense to add another metrics collection point (at least not at the scale we're currently at).

It also doesn't necessarily hurt to leave it in but I think at this point it just adds complexity for not much gain.